Calculator/Calculator/Controller/ViewController.swift

```diff
@@ -7,12 +7,156 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet weak var inputNumberLabel: UILabel!
+    @IBOutlet weak var inputOperatorLabel: UILabel!
+    @IBOutlet weak var allStackView: UIStackView!
+    @IBOutlet weak var scrollView: UIScrollView!
+    
+    private var resultString: String = ""
+    private var calculateSetting: Bool = true
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        inputNumberLabel.text = "0"
+        inputOperatorLabel.text = ""
+        
+        allStackView.arrangedSubviews.forEach{ $0.removeFromSuperview() }
+    }
+    
+    @IBAction func tapNumbersButton(_ sender: UIButton) {
+        if calculateSetting == true {
+            guard let inputNumber = sender.titleLabel?.text,
+                  let numberLabel = inputNumberLabel.text,
+                  let inputOperatorLabelText = inputOperatorLabel.text else { return }
+            
+            inputOperatorLabel.text = (allStackView.subviews.isEmpty) ? ("") : inputOperatorLabelText
+            inputNumberLabel.text = (numberLabel == "0") ? (inputNumber) : (numberLabel + inputNumber)
+        }
+    }
+    
+    @IBAction func tapSerialZeroButton(_ sender: UIButton) {
+        guard let inputNumber = sender.titleLabel?.text,
+              let numberLabel = inputNumberLabel.text else { return }
+        
+        inputNumberLabel.text = (numberLabel == "0") ? ("0") : (numberLabel + inputNumber)
+    }
+    
+    @IBAction func positiveNegativeButton(_ sender: UIButton) {
+        let minusSign = "-"
+        guard let numberLabel = inputNumberLabel.text else { return }
+        
+        if numberLabel != "0" {
+            inputNumberLabel.text? = (numberLabel.first.map(String.init) != minusSign) ? (minusSign + numberLabel) : (String(numberLabel.dropFirst(1)))
```

Comment:
```suggestion
            inputNumberLabel.text? = numberLabel.hasPrefix(minusSign) ? (minusSign + numberLabel) : (String(numberLabel.dropFirst(1)))
```

`hasPrefix`를 사용하면 보다 직관적일 것 같습니다.

Reply:
문자열을 검색하는 함수로 다양한 것들이 존재하네요..!! 건디 덕분에 새로운 함수를 배웠습니다!👍
해당 부분 수정하였습니다!